packages/cli/src/lib/snapshot/snapshot.ts

```diff
@@ -0,0 +1,37 @@
+import {ResourceSnapshotsModel} from '@coveord/platform-client';
+import ResourceSnapshots from '@coveord/platform-client/dist/definitions/resources/ResourceSnapshots/ResourceSnapshots';
```

Comment:
Is that the right import path ? It can't be imported from root of `@coveord/platform-client` ?

If so, would it make sense to open a PR/TODO for that in platform client project ?

Reply:
Correct, it is not available. I will open a PR
![image](https://user-images.githubusercontent.com/12199712/120200750-a9cd9100-c1f2-11eb-8b28-b89652614224.png)

Reply:
[CDX-366](https://coveord.atlassian.net/browse/CDX-366)

Reply:
@louis-bompart Here is the pull request : https://github.com/coveo/platform-client/pull/324